lib/tasks/change_allow_web_publish_to_false.rake

```diff
@@ -4,8 +4,33 @@ namespace :goodcity do
   desc 'change allow_web_publish to false if package.quantity = 0'
   task change_allow_web_publish_to_false: :environment do
     packages = Package.where("quantity = ? and allow_web_publish = ?", 0, true)
+
+    # code to create log for the rake
+    start_time = Time.now
+    File.open("rake_log.txt", "a+"){|f|
```

Comment:
Sure @steveyken , I has thought of it before but wasn't sure. Thanks for pointing.